ci/Makefile.base

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default settings for the CI/CD system.
+
+CLUSTER            ?= prow
+PROJECT            ?= knative-tests
+ZONE               ?= us-central1-f
+JOB_NAMESPACE      ?= test-pods
+KNATIVE_CONFIG     ?= config_knative.yaml
+SKIP_CONFIG_BACKUP ?=
+
+CI_DIR           ?= $(realpath $(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
feel like I can use a comment here

Reply:
Why now, but not in the original code? Just because of realpath()? :) Done.

Reply:
thanks, just think it would be useful for someone who is not super familiar with all of these commands